test/test_translator.py

```diff
@@ -84,27 +81,29 @@ def test_parse_incident() :
     ]
   }
 }
+    assert test_feature == valid_feature
 
+def test_parse_incident_success_no_data():
+  test_feature = icone_translator.parse_incident(None)
+  valid_feature=None
+  assert test_feature == valid_feature
 
-    assert test_feature == valid_feature
+def test_parse_incident_success_invalid_data():
```

Comment:
Instead of throwing a random error here we should be testing that the callback we included is called:
```
def test_parse_incident_success_invalid_data():
    test_var = 'a,b,c,d'
    callback = MagicMock()
    test_feature = icone_translator.parse_incident(
        test_var, callback_function=callback)
    assert callback.called
```

Also, we aren't testing a success case here, so let's remove that from the name.

Reply:
I updated the unit test and also checked  assert callback.called and test_feature == None